src/serac/physics/solid_mechanics.hpp

```diff
@@ -903,12 +903,6 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
 
     if (is_quasistatic_) {
       residual_with_bcs_ = buildQuasistaticOperator();
-
-      // the residual calculation uses the old stiffness matrix
-      // to help apply essential boundary conditions, so we
-      // compute J here to prime the pump for the first solve
-      residual_with_bcs_->GetGradient(displacement_);
-
```

Comment:
Do we want to remove this?  Seems unrelated to contact, maybe?

Reply:
It is unrelated -- just addressing the comment above: https://github.com/LLNL/serac/pull/982#discussion_r1307804104 .  It probably should be a different PR, but since I'm making other changes around this code path, I think it's OK to leave it in.